data/formats-data.js

```diff
@@ -1582,7 +1589,7 @@ exports.BattleFormatsData = {
 		tier: "BL",
 	},
 	omanyte: {
-		randomBattleMoves: ["shellsmash", "surf", "icebeam", "earthpower", "hiddenpowerelectric", "spikes", "toxicspikes", "stealthrock", "hydropump"],
+		randomBattleMoves: ["shellsmash", "icebeam", "earthpower", "spikes", "stealthrock", "hydropump"],
```

Comment:
ill take ur word that the rolls will work out with surf 
for knock off, see: ice shard shellder, but knock off is so great that it might have a spot anyway